x-pack/plugin/ccr/qa/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexSecurityIT.java

```diff
@@ -119,24 +119,28 @@ public void testFollowIndex() throws Exception {
 
             // User does have manage_follow_index index privilege on 'allowed' index,
             // but not read / monitor roles on 'disallowed' index:
-            e = expectThrows(ResponseException.class, () -> follow(unallowedIndex, allowedIndex));
+            e = expectThrows(ResponseException.class, () -> follow(client(), unallowedIndex, allowedIndex));
             assertThat(e.getMessage(), containsString("insufficient privileges to follow index [unallowed-index], " +
                 "privilege for action [indices:monitor/stats] is missing, " +
                 "privilege for action [indices:data/read/xpack/ccr/shard_changes] is missing"));
             // Verify that the follow index has not been created and no node tasks are running
             assertThat(indexExists(adminClient(), unallowedIndex), is(false));
             assertBusy(() -> assertThat(countCcrNodeTasks(), equalTo(0)));
 
-            e = expectThrows(ResponseException.class, () -> resumeFollow(unallowedIndex, unallowedIndex));
+            follow(adminClient(), unallowedIndex, unallowedIndex);
```

Comment:
Execute put follow api with admin client, so that we can check whether we fail with the right error when a normal user resumes following an unallowed index.